frontend/src/pages/LoginAndRegisterPage.module.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+body{
+    margin: 0;
+}
```

Comment:
good idea to set margins to 0, but this kind of thing should be done in a single global CSS file on all of the elements. Here is a snippet that you should add in the global.css 
```css
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```
Another thing I have to ask for is that you should move font imports into a global CSS file as well. You should always store configuration like fonts and setting default values like in the above snippet, inside a global CSS file, this helps managing it later on.